libraries/botbuilder-repo-utils/src/executor.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as R from 'remeda';
+import assert from 'assert';
+import async from 'async';
+import pmap from 'p-map';
+import { Workspace } from './workspace';
+
+// Describes a function signature for an executor that accepts a list of workspaces,
+// a task, and a concurrency parameter and executes the task against all workspaces.
+export type Executor = (
+    workspaces: Workspace[],
+    task: (workspace: Workspace) => Promise<boolean>,
+    concurrency: number
+) => Promise<boolean>;
```

Comment:
This is one of the main changes - execution now returns a boolean to indicate success. This is used to support continuing to execute tasks if one fails while still exiting with a non-zero status code.